capplets/Makefile.am

```diff
@@ -22,7 +29,7 @@ DIST_SUBDIRS = \
 	keyboard \
 	mouse \
 	network \
-	time-admin \
+	$(TIMEADMIN) \
```

Comment:
You must not define it conditionally in `DIST_SUBDIRS`, so that tarballs & co are properly created.  Just remove this change, having it conditional in `SUBDIRS` is enough to do what you want.  You can [check the documentation](https://www.gnu.org/software/automake/manual/automake.html#Conditional-Subdirectories) which is surprisingly clear :)

Actually, `DIST_SUBDIRS` doesn't really seem useful at all, but let's not touch this in this PR as it's unrelated.